Update getting-started-with-ballerina.md (#3674)
ballerina-platform/ballerina-dev-website: learn/getting-started/getting-started-with-ballerina.md

In this code:

> - The first line is an import statement that makes the functionality in the [`ballerina/io`](https://lib.ballerina.io/ballerina/io/latest) module available to your program. This module contains functions to write to the console, read from the console, and perform read/write operations on the files.

Author: Or
```suggestion
- The first line is an import statement that makes the capabilities of the [`ballerina/io`](https://lib.ballerina.io/ballerina/io/latest) module available to your program. This module contains functions to write to the console, read from the console, and perform read/write operations on the files.
```

Reviewer: @sameerajayasoma - WDYT?
